recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "4.2" %}
```

Comment:
So, I actually need `3.7.1` for some things. Is it ok if we start with that version? It is always possible to bump the version in the feedstock.